src/mumble/AudioOutput.cpp

```diff
@@ -349,6 +350,10 @@ void AudioOutput::initializeMixer(const unsigned int *chanmasks, bool forceheadp
 	}
 	iSampleSize = static_cast< int >(iChannels * ((eSampleFormat == SampleFloat) ? sizeof(float) : sizeof(short)));
 	qWarning("AudioOutput: Initialized %d channel %d hz mixer", iChannels, iMixerFreq);
+	
+	if (Global::get().s.bPositionalAudio && iChannels == 1)
+	    Global::get().l->log(Log::CriticalError, tr("Positional Audio will not work on mono output devices!"));
```

Comment:
```suggestion
		Global::get().l->log(Log::CriticalError, tr("Positional audio cannot work with mono output devices!"));
```

Reply:
I would also change `Log::CriticalError` with `Log::Warning`.

Reply:
And one more thing: We should probably use `GLobal::get().l->logOrDefer` as I am not sure that it is guaranteed that the logging system is _always_ initalized before this code runs.